internal/builders/container/README.md

```diff
@@ -321,6 +322,86 @@ Certificate issuer URL:  https://token.actions.githubusercontent.com
+as part of [Admission Control](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+```yaml
+apiVersion: policy.sigstore.dev/v1alpha1
+kind: ClusterImagePolicy
+metadata:
+  name: image-is-signed-by-github-actions
+spec:
+  images:
+    # Matches all versions of the actions-test image.
+    # NOTE: policy-controller mutates pods to use a digest even if originally
+    # specified by tag.
+    - glob: "ghcr.io/ianlewis/actions-test@*"
+  authorities:
+    - keyless:
+        # Signed by the public Fulcio certificate authority
+        url: https://fulcio.sigstore.dev
+        identities:
+          # Matches the Github Actions OIDC issuer
+          - issuer: https://token.actions.githubusercontent.com
+            # Matches the reusable workflow's signing identity.
+            # TODO: update after GA
+            subjectRegExp: "^https://github.com/ianlewis/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/heads/409-feature-add-generic-container-workflow$"
+      attestations:
+        - name: must-have-slsa
+          predicateType: slsaprovenance
+          policy:
+            type: cue
+            data: |
+              // The predicateType field must match this string
+              predicateType: "https://slsa.dev/provenance/v0.2"
+
+              predicate: {
+                // This condition verifies that the builder is the builder we
+                // expect and trust. The following condition can be used
+                // unmodified. It verifies that the builder is the container
+                // workflow.
+                builder: {
+                  // TODO: update after GA
+                  // id: =~"^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$"
+                  id: =~"^https://github.com/ianlewis/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/heads/409-feature-add-generic-container-workflow$"
+                }
+                invocation: {
+                  configSource: {
+                    // This condition verifies the entrypoint of the workflow.
+                    // Replace with the relative path to your workflow in your
+                    // repository.
+                    entryPoint: ".github/workflows/generic-container.yml"
+
+                    // This condition verifies that the image was generated from
+                    // the source repository we expect. Replace this with your
+                    // repository.
+                    uri: =~"^git\\+https://github.com/ianlewis/actions-test@refs/tags/v[0-9]+.[0-9]+.[0-9]+$"
+                  }
+                }
+              }
+```
+
+When applied the `ClusterImagePolicy` will be evaluated when a `Pod` is
+created. If the `Pod` fulfills the policy conditions then the `Pod` can be
+created. If the `Pod` does not fulfill one or more of the policy conditions
```

Comment:
This behaviour can be configured whenever a Pod doesn't match to any policy to deny, allow or warn:

```
In addition to that, the policy controller offers a configurable behavior defining whether to allow, deny or warn whenever an image does not match a policy. This behavior can be configured using the config-policy-controller ConfigMap created under the release namespace, and by adding an entry with the property no-match-policy and its value warn|allow|deny. By default, any image that does not match a policy is rejected whenever no-match-policy is not configured in the ConfigMap.
```

You can find additional info here: https://github.com/sigstore/docs/blob/main/content/en/policy-controller/overview.md. Perhaps you could simply add the following https://github.com/sigstore/docs/blob/main/content/en/policy-controller/overview.md#admission-of-images here.

Reply:
> This behaviour can be configured whenever a Pod doesn't match to any policy to deny, allow or warn:
> 
> ```
> In addition to that, the policy controller offers a configurable behavior defining whether to allow, deny or warn whenever an image does not match a policy. This behavior can be configured using the config-policy-controller ConfigMap created under the release namespace, and by adding an entry with the property no-match-policy and its value warn|allow|deny. By default, any image that does not match a policy is rejected whenever no-match-policy is not configured in the ConfigMap.
> ```
> 
> You can find additional info here: https://github.com/sigstore/docs/blob/main/content/en/policy-controller/overview.md. Perhaps you could simply add the following https://github.com/sigstore/docs/blob/main/content/en/policy-controller/overview.md#admission-of-images here.

Thanks for the feedback! The content you linked to is the same as on the website right? Perhaps that's a better link?
https://docs.sigstore.dev/policy-controller/overview/#admission-of-images

Reply:
It is related to the latest available version. I'm gonna solve the problem on the website, it should get those changes in there.

Reply:
Yes, I'll use [https://docs.sigstore.dev/policy-controller](https://docs.sigstore.dev/policy-controller) for any related info about the policy-controller.

Reply:
Cool thanks.

Reply:
Fixed now in https://docs.sigstore.dev/policy-controller/overview/#admission-of-images!